### DISCUSSION TITLE
Unable to edit blueprint

### DISCUSSION THREAD

**Discussion by lasseron** [statamic/cms#6766] [Troubleshooting] [answered]
Our site initially on Laravel 6, Statamic 3.2 for a year. 
After upgrading to Laravel 8, Statamic 3.3 everything worked smoothly until we ran "composer update" one day.

Mouse click on existing field titles or "Link existing" and "Create field" buttons now do absolutely nothing on "Edit blueprint" page.
The same behaviour is appears with some other buttons such as "Publish" button when using revisions. 

What we have investigated or tried without any luck:

- There are no JS errors on console. 
- Vue components such as <favorite-creator> or <favorite-creator> seem to render without any issue. Dragging field order works.
- Downgraded Statamic to 3.3.0 and upgraded to 3.3.7. 
- Downgraded various packages.
- Upgraded PHP from 7.4 to 8.0 and 8.1, run composer update. 
- We have emptied Laravel route file.
- We have played with various Statamic configuration options. 
- Cleared Laravel cache from shell, Statamic cache from both shell and from the UI.

Would be greatful for any hint on where to check next!

**Comment by jasonvarga** [comment, marked as answer]
Try deleting `public/vendor/statamic/cp` and running `composer install` again. Then hard refresh the browser. Sounds like maybe you have outdated js assets somehow.

**Reply by lasseron** [reply]
Thank you, this worked like a charm! 
Turns out we had accidentally removed Statamic from "post-autoload-dump" section of composer.json while upgrading Laravel.
